samcli/commands/_utils/options.py

```diff
@@ -788,3 +789,21 @@ def _space_separated_list_func_type(value):
 
 
 _space_separated_list_func_type.__name__ = "LIST"
+
+
+def generate_next_command_recommendation(command_tuples: List[tuple]):
+    """
+    Generates a message containing some suggested commands to run next.
+
+    :type command_tuples: list
+    :param command_tuples: list of tuples containing the command with their respective description
+    """
+    template = """
+Commands you can use next:
+================================
```

Comment:
Match existing look:

```suggestion
Commands you can use next
=========================
```

I'm not suggesting the current UI is the most ideal, but it's less friction to do things in separate steps: one that only augments without changing existing behavior, and another that addresses UI shortcomings in a more encompassing manner across the SAM CLI.

Reply:
That makes sense, Done!

Reply:
Done!